wordpress-mobile/WordPress-Android
# Issue/62 posts pages list search

Fixes #62. Refer to #5730 for background.

To test:
* Sign into a self-hosted site
* Verify that search is not available in the posts and pages lists
* Sign into a Jetpack and/or .com site
* Verify that search is available in the posts and pages lists
* Begin typing and verify that the progress indicator is displayed
* Wait for results (1.5s delay in searching) and verify progress indicator is hidden
* Type more before search results are shown to verify network calls are continuously delayed until user stops typing for 1.5s

cc @oguzkocer
